### PR TITLE
Add a python2.7 base image.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
Ref #24

File: WORKSPACE [hunk 1]
```diff
@@ -69,6 +69,17 @@ new_http_archive(
    type = "tgz",
 )
 
+# For Python
+http_file(
+   name = "libpython27",
+   url = "http://ftp.us.debian.org/debian/pool/main/p/python2.7/libpython2.7-minimal_2.7.9-2+deb8u1_amd64.deb",
```
Comment (mattmoor) [inline WORKSPACE]: sha256 :)

File: cloudbuild.yaml [hunk 2]
```diff
@@ -14,9 +14,15 @@ steps:
...
+  args: ['run', '//python2.7:python27']
+- name: gcr.io/cloud-builders/docker
+  args: ['tag', 'bazel/python2.7:python27', 'gcr.io/$PROJECT_ID/python2.7:latest']
+
 # The resulting images to push to GCR.
 images: [
   'gcr.io/$PROJECT_ID/base:latest',
   'gcr.io/$PROJECT_ID/java:latest',
   'gcr.io/$PROJECT_ID/java/jetty:latest',
+  'gcr.io/$PROJECT_ID/python2.7:latest'
```
Comment (mattmoor) [inline cloudbuild.yaml]: nit: we put 2.7 in the name for python, but not for java.  I understand Python's weirder, but we should consider how we might resolve this.

**Comment by dlorenc** [comment]
> You plan to add samples separately?

Yeah I'll get some samples in another PR.

**Comment by mattmoor** [comment]
We discussed `python2.7` offline, so I think the only outstanding comment is adding `sha256`s.

**Comment by dlorenc** [comment]
> We discussed python2.7 offline, so I think the only outstanding comment is adding sha256s.

Done.

**Pull request merged**